specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1920,6 +2240,339 @@
+          }
+        },
+        "radiiGeometry": {
+          "$ref": "../../../Common/preview/1.0/common.json#/definitions/GeoJsonGeometry",
+          "description": "GeoJSON object. Displayed when radiiGeometry=true in request. Describes the outline of the wind radius quadrants."
+        }
+      }
+    },
+    "RadiusSector": {
+      "type": "object",
+      "description": "Bearing 0–90 = NE quadrant; 90–180 = SE quadrant; 180–270 = SW quadrant; 270–360 = NW quadrant.",
+      "properties": {
+        "beginBearing": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Bearing, in degrees, of the beginning of the quadrant."
+        },
+        "endBearing": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Bearing, in degrees, of the end of the quadrant."
+        },
+        "range": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The radius of the quadrant, in nautical miles."
+        }
+      }
+    },
+    "StormLocationsResponse": {
+      "type": "object",
+      "description": "Locations for an individual government-issued storm",
+      "properties": {
+        "results": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LocationResponse"
+          },
+          "description": "Locations for an individual government-issued storm"
+        }
+      }
+    },
+    "LocationResponse": {
+      "type": "object",
+      "description": "Location for an individual Government-issued storm",
+      "properties": {
+        "dateTime": {
+          "type": "string",
+          "description": "Datetime the forecast is valid, displayed in ISO8601 format."
+        },
+        "location": {
+          "$ref": "../../../Common/preview/1.0/common.json#/definitions/CoordinatesPair",
+          "description": "Coordinates of the storm"
+        },
+        "maxWindGust": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Maximum wind gust speed associated with the storm. May be NULL."
+        },
+        "sustainedWind": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Maximum sustained wind speed associated with the storm. May be NULL."
+        },
+        "minimumPressure": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Minimum pressure associated with the storm. May be NULL."
+        },
+        "movement": {
+          "$ref": "#/definitions/WindDetails",
+          "description": "True when the depression is classified as a subtropical cyclone."
+        },
+        "status": {
+          "type": "string",
+          "description": "Storm status, in English."
+        },
+        "isSubtropical": {
+          "type": "boolean",
+          "description": "Automated Tropical Cyclone Forecasting (ATCF) storm ID. May be NULL. If NULL, this parameter will not be displayed."
```

Comment:
True when the depression is classified as a subtropical cyclone.

Reply:
Good catch! Updated